04_Callback/readme.md

```diff
@@ -36,7 +36,7 @@ _nameEdit.tsx_
 
 ```diff
 import * as React from 'react';
-import {Fragment} from 'react';
+import { Fragment } from 'react';
```

Comment:
I think we can remove this import since we are not using <Fragment>, must the helper, can you give a try? thanks